<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CatalogoActivoFijoGUI</class>
 <widget class="QWidget" name="CatalogoActivoFijoGUI">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>550</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>640</width>
    <height>550</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Ingreso de epecificaciones al catalogo de activo fijo</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>601</width>
     <height>441</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Activo Fijo</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>571</width>
       <height>131</height>
      </rect>
     </property>
     <property name="title">
      <string>Asigne la cuenta de inventario y clase</string>
     </property>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>46</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>Cuenta:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>46</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>Clase:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="nombreClase">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>95</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="maxLength">
       <number>3</number>
      </property>
     </widget>
     <widget class="QLineEdit" name="cuentaAsignada">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>60</y>
        <width>191</width>
        <height>21</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="numeroCuenta">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>30</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="maxLength">
       <number>5</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>30</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Especifico:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="numeroEspecifico">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>30</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="maxLength">
       <number>5</number>
      </property>
     </widget>
     <widget class="QLineEdit" name="especifico">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>60</y>
        <width>201</width>
        <height>20</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="Line" name="line_3">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>20</y>
        <width>20</width>
        <height>91</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>240</y>
       <width>571</width>
       <height>121</height>
      </rect>
     </property>
     <property name="title">
      <string>Datos Generales:</string>
     </property>
     <widget class="QLineEdit" name="descripcion">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>30</y>
        <width>451</width>
        <height>20</height>
       </rect>
      </property>
      <property name="maxLength">
       <number>60</number>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Descripci√≥n:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>46</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>Marca:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>70</y>
        <width>46</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>Modelo:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="marca">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>70</y>
        <width>201</width>
        <height>20</height>
       </rect>
      </property>
      <property name="maxLength">
       <number>30</number>
      </property>
     </widget>
     <widget class="QLineEdit" name="modelo">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>70</y>
        <width>171</width>
        <height>20</height>
       </rect>
      </property>
      <property name="maxLength">
       <number>15</number>
      </property>
     </widget>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>571</width>
       <height>21</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>271</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Ingreso de Especifciaciones de Activo Fijo al catalogo.</string>
     </property>
    </widget>
    <widget class="QToolButton" name="botonEliminar">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>10</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Eliminar</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../ContablesPrincipal.qrc">
       <normaloff>:/iconos/editdelete.png</normaloff>:/iconos/editdelete.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </widget>
    <widget class="QToolButton" name="botonNuevo">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>10</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Nuevo</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../ContablesPrincipal.qrc">
       <normaloff>:/iconos/filenew.png</normaloff>:/iconos/filenew.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </widget>
    <widget class="QToolButton" name="botonModificar">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>10</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Modificar</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../ContablesPrincipal.qrc">
       <normaloff>:/iconos/trabajador.png</normaloff>:/iconos/trabajador.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </widget>
    <widget class="QToolButton" name="botonGuardar">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>10</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Guardar</string>
     </property>
     <property name="whatsThis">
      <string extracomment="Guardar"/>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../ContablesPrincipal.qrc">
       <normaloff>:/images/save.png</normaloff>:/images/save.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </widget>
    <widget class="QToolButton" name="ayudaActivoFijo">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>10</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../ContablesPrincipal.qrc">
       <normaloff>:/images/help.png</normaloff>:/images/help.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>Material Bibliografico</string>
    </attribute>
    <widget class="Line" name="line_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>561</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>261</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Ingreso de especificaciones de Material Bibliografico</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>551</width>
       <height>81</height>
      </rect>
     </property>
     <property name="title">
      <string>Ingrese la clase: </string>
     </property>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>46</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Cuenta:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>20</y>
        <width>46</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>Clase:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>50</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Especifico:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="claseBiblio">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>20</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="maxLength">
       <number>3</number>
      </property>
     </widget>
     <widget class="QLineEdit" name="cuentaBiblio">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>20</y>
        <width>231</width>
        <height>20</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="especificoBiblio">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>50</y>
        <width>231</width>
        <height>20</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>220</y>
       <width>551</width>
       <height>81</height>
      </rect>
     </property>
     <property name="title">
      <string>Datos Generales autor:</string>
     </property>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>46</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>Nombre:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>46</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Apellidos:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="apellidos">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>50</y>
        <width>311</width>
        <height>20</height>
       </rect>
      </property>
      <property name="maxLength">
       <number>35</number>
      </property>
     </widget>
     <widget class="QLineEdit" name="nombreAutor">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>20</y>
        <width>311</width>
        <height>20</height>
       </rect>
      </property>
      <property name="maxLength">
       <number>50</number>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>310</y>
       <width>551</width>
       <height>91</height>
      </rect>
     </property>
     <property name="title">
      <string>Datos Generales</string>
     </property>
     <widget class="QLineEdit" name="tituloBiblio">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>50</y>
        <width>431</width>
        <height>20</height>
       </rect>
      </property>
      <property name="maxLength">
       <number>85</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>50</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Titulo:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>46</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>ISBN:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="isbn">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>20</y>
        <width>231</width>
        <height>20</height>
       </rect>
      </property>
      <property name="maxLength">
       <number>25</number>
      </property>
     </widget>
    </widget>
    <widget class="QToolButton" name="botonEliminarBiblio">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>10</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../ContablesPrincipal.qrc">
       <normaloff>:/iconos/editdelete.png</normaloff>:/iconos/editdelete.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </widget>
    <widget class="QToolButton" name="botonNuevoBiblio">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>10</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../ContablesPrincipal.qrc">
       <normaloff>:/iconos/filenew.png</normaloff>:/iconos/filenew.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </widget>
    <widget class="QToolButton" name="botonModificaBiblio">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>10</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../ContablesPrincipal.qrc">
       <normaloff>:/iconos/trabajador.png</normaloff>:/iconos/trabajador.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </widget>
    <widget class="QToolButton" name="botonGuardarBiblio">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>10</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../ContablesPrincipal.qrc">
       <normaloff>:/images/save.png</normaloff>:/images/save.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButton">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>10</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../ContablesPrincipal.qrc">
       <normaloff>:/images/help.png</normaloff>:/images/help.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>551</width>
       <height>61</height>
      </rect>
     </property>
     <property name="title">
      <string>Descripcion de la Clase:</string>
     </property>
     <widget class="QLineEdit" name="descripcionBiblio">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>511</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Unidad:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Departamento:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nombreUnidad">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>20</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="nombreDpto">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>50</y>
     <width>261</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>20</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../ContablesPrincipal.qrc">:/iconos/kedit.png</pixmap>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>20</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="plainText">
    <string>Cree, modifique o elimine especificaciones de los bienes inventariables.</string>
   </property>
   <property name="backgroundVisible">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>numeroCuenta</tabstop>
  <tabstop>nombreClase</tabstop>
  <tabstop>numeroEspecifico</tabstop>
  <tabstop>descripcion</tabstop>
  <tabstop>marca</tabstop>
  <tabstop>modelo</tabstop>
  <tabstop>botonGuardar</tabstop>
  <tabstop>botonModificar</tabstop>
  <tabstop>botonNuevo</tabstop>
  <tabstop>botonEliminar</tabstop>
  <tabstop>ayudaActivoFijo</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>claseBiblio</tabstop>
  <tabstop>descripcionBiblio</tabstop>
  <tabstop>nombreAutor</tabstop>
  <tabstop>apellidos</tabstop>
  <tabstop>isbn</tabstop>
  <tabstop>tituloBiblio</tabstop>
  <tabstop>botonGuardarBiblio</tabstop>
  <tabstop>botonModificaBiblio</tabstop>
  <tabstop>botonNuevoBiblio</tabstop>
  <tabstop>botonEliminarBiblio</tabstop>
  <tabstop>toolButton</tabstop>
  <tabstop>especificoBiblio</tabstop>
  <tabstop>cuentaBiblio</tabstop>
  <tabstop>nombreUnidad</tabstop>
  <tabstop>nombreDpto</tabstop>
  <tabstop>plainTextEdit</tabstop>
  <tabstop>cuentaAsignada</tabstop>
  <tabstop>especifico</tabstop>
 </tabstops>
 <resources>
  <include location="../ContablesPrincipal.qrc"/>
 </resources>
 <connections/>
</ui>
